### DISCUSSION TITLE
Using measured laser profile

### DISCUSSION THREAD

**Discussion by berceanu** [BLAST-WarpX/warpx#4349] [Q&A] [answered]
Does `WarpX` have support for [lasy](https://github.com/LASY-org/lasy)? I am asking because I would like to use a measured laser pulse in a PIC simulation.

**Comment by RemiLehe** [comment, marked as answer]
Yes, we recently added support for `lasy`. See for instance this example file: https://github.com/ECP-WarpX/WarpX/blob/development/Examples/Tests/laser_injection_from_file/inputs.3d_test
